metrotranscom/doorway
# Fix/feedback quick fixes - Change Doorway Icon out & Add meta tags & fix FAQ sections

# Pull Request Template

## Issue Overview

This PR addresses #issue

## Description
Quick fixes for:
- adding in meta info on the new pages, updating the old ones
- removing the doorway emoji from the site in exchange for "Doorway"
- fix the sections on the FAQ page, so all of the topic headings show up on the left listing.

## How Can This Be Tested/Reviewed?
View code changes for translations & meta info. You can build & run the Partners site and verify the header meta info via Chrome Inspect. Double check that you don't see the 🚪emoji anywhere anymore...but rather "Doorway".

You can verify that the headers are now showing on the FAQ page:
before: 
![image](https://github.com/metrotranscom/doorway/assets/12400715/0c2b601f-e6d8-4665-9e54-dee1f6af46da)

after:
![image](https://github.com/metrotranscom/doorway/assets/12400715/cf569676-e5cb-47e9-ba91-bf5cb37c8002)


## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
